### ISSUE TITLE
open calendar.ics files

### ISSUE THREAD

**Issue by lopippo** [itchyny/calendar.vim#137]
Greetings,

I would find most useful a feature that would allow to open a <file>.ics file in vim using this plugin that seems powerful enough to just do this. 

Do you think this is feasible ?

Sincerely,
lopippo

**Comment by matthewdeancooper** [comment]
+1

This would be an awesome feature. 

IMO it would be better if we could just supply a URL to a live ics file that is auto synced down to calendar.vim

**Comment by manvhah** [comment]
+1 this would be most useful!

**Comment by Ginner** [comment]
It would be very nice indeed! It would also solve integration with iCloud and others. To me it's crucial, I cannot use it unless it integrates with the services I'm using, and I can only see it integrating with google.

Integration with a service agnostic file format like .ics would be a game changer.

**Comment by dtrckd** [comment]
+1,
It would be great and allow to sync with the Thunderbird's calendar.

**Comment by rickalex21** [comment]
I agree. There's a python module ics files, it would make things much easier but I have not tried it personally myself.

**Comment by OfekiAlm** [comment]
Is anything going to work on this feature?
